### DISCUSSION TITLE
Logs

### DISCUSSION THREAD

**Discussion by RagibHasin** [ramensoftware/windhawk#2] [Q&A] [answered]
From where the logs from various mods can be viewed? It seems that the "Taskbar Clock Customization" mod is broken on Windows 11 22000.556 an no log can be found.

**Comment by m417z** [comment, marked as answer]
You have two options:

**Via Windhawk**

* Disable the Taskbar Clock Customization mod.
* Fork the mod.
* Compile the mod.
* Enable logging.
* Click on Show Log Output.
* You can see the log at the bottom of the window.

**Via a third party tool**

* Download and run [DebugView](https://docs.microsoft.com/en-us/sysinternals/downloads/debugview) or a similar tool.
* Open regedit.
* Navigate to `HKEY_LOCAL_MACHINE\SOFTWARE\Windhawk\Engine\Mods\taskbar-clock-customization`
* Change `LoggingEnabled` from `0` to `1`.
* You can see the logs in your tool. You can filter for Windhawk logs by filtering for lines containing `[WH]`.

**Reply by m417z** [reply]
Update: In Windhawk v1.0, logs can be enabled and viewed in the Advanced tab.

![image](https://user-images.githubusercontent.com/4129781/209521857-aa0175d8-d730-4b5d-9a97-11db03f89092.png)

**Comment by GiuseWolf1** [comment]
Bug Report – Translucent Windows Mod​  (Windhawk 1.6.1)
Mod: Translucent Windows Version: 1.6.3 (or current)
Operating System: Windows 11 23H2 (or your current version)

Issues encountered:

1. When renaming a file in File Explorer, the text field becomes completely black or invisible.
2. With HDR enabled, some windows become overly transparent or unreadable.
3. Taskbar icons sometimes become unresponsive when multiple windows are minimized.
4. Context menus and system dialogs occasionally become illegible due to excessive transparency.

Troubleshooting attempts:

-Disabled HDR
-Added explorer.exe to the “Excluded Processes” list
-Disabled “Text Alpha Blending”
-Restarted File Explorer after each change

Personal note:

The mod delivers a beautiful visual effect, but it introduces several usability issues across the Windows GUI. It would be helpful to have an option to apply the effect only to non-system windows or to limit its scope more precisely.
